src/e2e/puppeteer/helpers/dragAndDropThought.ts

```diff
@@ -0,0 +1,35 @@
+import { Page } from 'puppeteer'
+import getEditable from './getEditable'
+
+/** PErforms Drag and Drop functionality on a hought in Puppeteer browser. */
+const dragAndDropThought = async (page: Page, sourceValue: string, destValue: string) => {
+  const sourceElement = await getEditable(page, sourceValue)
+  const destElement = await getEditable(page, destValue)
+
+  const dragStart = await sourceElement.boundingBox()
+  const dragEnd = await destElement.boundingBox()
+
+  if (!dragStart || !dragEnd) {
+    return
+  }
+
+  const yOffset = 20 // Adding offset to drop destination y position so it is dragged exactly on top of it. Right now it fall short just above the drop target
+
+  // Calculate center positions of the elements
```

Comment:
The centering calculations are a bit superfluous since we could just drop it on the top, left corner, but I guess it doesn't hurt.

Reply:
So should I change it or keep as it is?

Reply:
You can keep it as is, I'm just pointing out that it was unnecessary.